### PR TITLE
Fork test

### PR DESCRIPTION
**Pull request by frankplus**
Questo è un test di pull request da una fork

### CODE REVIEW

File: app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
  * The Activity useful for making photos
  */
 public class CameraActivity extends AppCompatActivity {
-
+    //THIS IS A FORK TEST
```
Comment (PrandiniUniPD) [inline app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java:27]: Yeah! This is a test of an inline comment

Comment (PrandiniUniPD) [inline app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java:27]: This is a test of a conversation

**Review by PrandiniUniPD — COMMENTED** [review]
This is a test of a review

File: app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java [hunk 2]
```diff
@@ -28,6 +28,8 @@
     private CameraKitView cameraKitView;
     private static String orientationResult;
 
+    //THIS IS A SECOND COMMIT
+
```
Comment (PrandiniUniPD) [inline app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java:32]: New commit, new test of inline comment!

**Review by PrandiniUniPD — COMMENTED** [review]
Hurray!

File: app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java (hunk 1 above)
Comment (frankplus) [reply app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java:27]: Nice thank you

**Comment by PrandiniUniPD** [comment]
Test: if we close this pull request, is it remain in the history of the repo?

**Comment by PrandiniUniPD** [comment]
> Test: if we close this pull request, is it remain in the history of the repo?

Yeah! it remains

**Comment by PrandiniUniPD** [comment]
Test of a comment of an unmodified line.

https://github.com/frankplus/OCRCamera/blob/0a0f7edc74013e92cd2cfb5ef4aea19ba002b84b/app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java#L1
This is an unmodified line.
There is a permalink for each line.

**Pull request closed without merging**